scripts/loud.coffee

```diff
@@ -20,7 +20,12 @@
 module.exports = (robot) ->
   robot.hear /^([A-Z"][A-Z0-9 .,'"()\?!&%$#@+-]+)$/, (res) ->
     # Pick a loud from the stored list and say it. Skip if there are no louds.
-    old_loud = res.random(robot.brain.get('louds'))
+    old_loud = getRandomLoud()
+    while true
+      if old_loud not in robot.brain.get('louds_banned')
```

Comment:
I'd rather not loop on query, just trust that a previous ban has evicted it or the next one will